src/Servers/Kestrel/shared/test/HttpParsingData.cs

```diff
@@ -195,6 +203,12 @@ public static IEnumerable<string> RequestLineInvalidData
                     "CUSTOM / HTTP/1.1\n",
                     "CUSTOM / HTTP/1.0\rA\n",
                     "CUSTOM / HTTP/1.1\ra\n",
+                    "CUSTOM  / HTTP/1.1\r\n",
+                    "CUSTOM   / HTTP/1.1\r\n",
+                    "CUSTOM  /  HTTP/1.1\r\n",
+                    "CUSTOM   /   HTTP/1.1\r\n",
+                    "CUSTOM / HTTP/1.1 \r\n",
+                    "CUSTOM / HTTP/1.1  \r\n",
```

Comment:
Out of curiosity, do we know if HTTP.sys would accept any of the above invalid requests?